lib/nsxv.py

```diff
@@ -326,14 +329,25 @@ def __str__(self):
     for s in services:
       service = '%s%s' % (service, s)
 
+    #applied To
+    appliedToList = ''
+    if self.applied_to:
+      appliedToList = '<appliedToList>'
+      appliedTo = '%s%s%s' % (_XML_TABLE.get('appliedToStart'),
+                              self.applied_to,
+                              _XML_TABLE.get('appliedToEnd'))
+      appliedToList = '%s%s' %(appliedToList, appliedTo)
+      appliedToList = '%s%s' %(appliedToList, '</appliedToList>')
+
```

Comment:
too many blank lines here, please remove one. Two blank lines come before a class, one is allowed elsewhere